### ISSUE TITLE
How necessary is it to add Pentium 4 today?

### ISSUE THREAD

**Issue by BridgeHeadland** [joncampbell123/dosbox-x#4730]
### Question

Personally, I think it will be necessary if Pentium 2 and Pentium 3 are fully finished, I don't think P2 and P3 are complete yet, because Windows NT generally crashes if one of these is set as the CPU type value, whether the format is FAT16, FAT32 or NTFS. Now that I understand that WNT cannot be run in real DOS, although they can be run in DOSBox-X, in the form of dual-boot or multi-boot options, which for some reason can be run in DOSBox-X.
Because support for Windows ME, which is the last real DOS-based operating system, ended on December 31, 2003, when the Pentium M was still the very latest CPU type, I see no reason for Pentium D or later to be added, unless it becomes relevant to focus on support for Windows NT operating systems, until the limit is reached, and if so I believe it is, that the prompt will not be able to continue the installation of Windows Vista from step 2, because the installation and setup files may not be able to run in DOS mode (time will tell), probably won't need Intel i10 or higher, since Intel i9 was the latest CPU type when the last update of Windows XP occurred, May 26, 2019 - I am unsure whether Windows Server 2003 and Windows Home Server are still updated or not.

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by tazok81** [comment]
As wrote @BridgeHeadland, p2, p3 cpu emulation make win9.x to crash sooner or later. If they work then i feel faster emulation. But, only p1 works fine. That's mine experience.

**Comment by BridgeHeadland** [comment]
@Torinde 

> If Vista installation can't be started from a guest DOS, then you can start it from guest Win9x/XP - https://github.com/joncampbell123/dosbox-x/issues/3538#issuecomment-1712944713?

Yes, I have tried but it crashed. I've done more research and found that Windows XP is the last operating system to use NTLDR and that it was replaced by Windows Boot Manager (BOOTMGR) starting with Windows Vista. Now, I don't know if the Windows Boot Manager is included in DOSBox-X, or has the ability to be included, but I doubt it, although I could be wrong. Even if DOSBox-X won't add, or has Windows Boot Manager added already, it may be that Windows Vista can be installed 100%, and run in DOSBox-X, under NTLDR, only that it will then behave in a limited way, perhaps like the last Windows Longhorn version that is based on Windows XP (Which is apparently bootable from NTDLR), but that's just pure guesswork on my part, and nothing I can confirm.
If it so that DOSBox-X can only emulate NTDLR, and not Windows Boot Manager, and NTLDR is not DOS (I think), I would personally call DOSBox-X for NTLDR-Box+, and by + I mean that it works so for me that DOSBox-X is an emulator that emulates NTDLR which behaves like MS-DOS in general, in addition to itself, with FAT12 as file system, since you cannot run Scandisk in DOSBox-X, unless you do not run it in some image file with a better file system format.
